cli/tests/environment-pull.bats

```diff
@@ -297,6 +297,17 @@ function add_insecure_package() {
   assert_success
 }
 
+# bats test_tags=pull:unsupported
+# pulling an environment without packages for the current platform
+@test "pull environment inside the same environment with '--force' flag" {
+  update_dummy_env "owner" "name"
+  make_incompatible "owner" "name"
```

Comment:
`make_incompatible` is only needed when testing how we handle incompatible environments specifically
```suggestion
```

Reply:
Ack & resolved.